diff -up cryptography-3.2.1/src/_cffi_src/openssl/rsa.py.omv~ cryptography-3.2.1/src/_cffi_src/openssl/rsa.py
--- cryptography-3.2.1/src/_cffi_src/openssl/rsa.py.omv~	2020-11-04 18:55:01.377726775 +0100
+++ cryptography-3.2.1/src/_cffi_src/openssl/rsa.py	2020-11-04 18:57:41.587952259 +0100
@@ -51,14 +51,14 @@ int EVP_PKEY_CTX_set_rsa_oaep_md(EVP_PKE
 CUSTOMIZATIONS = """
 static const long Cryptography_HAS_PSS_PADDING = 1;
 
-#if defined(EVP_PKEY_CTX_set_rsa_oaep_md)
+#if 1 /*defined(EVP_PKEY_CTX_set_rsa_oaep_md)*/
 static const long Cryptography_HAS_RSA_OAEP_MD = 1;
 #else
 static const long Cryptography_HAS_RSA_OAEP_MD = 0;
-int (*EVP_PKEY_CTX_set_rsa_oaep_md)(EVP_PKEY_CTX *, EVP_MD *) = NULL;
+int (*EVP_PKEY_CTX_set_rsa_oaep_md)(EVP_PKEY_CTX *, const EVP_MD *) = NULL;
 #endif
 
-#if defined(EVP_PKEY_CTX_set0_rsa_oaep_label)
+#if 1 /* defined(EVP_PKEY_CTX_set0_rsa_oaep_label) */
 static const long Cryptography_HAS_RSA_OAEP_LABEL = 1;
 #else
 static const long Cryptography_HAS_RSA_OAEP_LABEL = 0;
